features/api/authorization-for-admin.feature

```diff
@@ -50,4 +53,15 @@ Feature: Book API Authorization for Admin Role
       | title       | author      |
       | Admin Book  | Admin Author |  
     When I send a "DELETE" request to "/api/books/{stored-id}"
-    Then the response status code should be 200
\ No newline at end of file
+    Then the response status code should be 200
+
+  Scenario: Admin can retrieve an existing book
```

Comment:
this is wrong , you need to create new feature file called book-retrieval.feature and do this scenarios there. not here